PetAdoption-iOS/PetAdoption-iOS/LicensesViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LicensesViewController.swift
+//  PetAdoption-iOS
+//
+//  Created by Amir Fleminger on 7/3/18.
+//  Copyright © 2018 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+
+struct item {
```

Comment:
The only criticism I would have is that the convention is to capitalize classes and structs
> Whenever you define a new structure or class, you define a new Swift type. Give types UpperCamelCase names (such as SomeStructure and SomeClass here) to match the capitalization of standard Swift types (such as String, Int, and Bool). Give properties and methods lowerCamelCase names (such as frameRate and incrementCount) to differentiate them from type names.

> https://docs.swift.org/swift-book/LanguageGuide/ClassesAndStructures.html

Other than that, looks good!